duckduckgo/iOS
# Home row CTA Copy experiment

Task/Issue URL: 
- https://app.asana.com/0/856498667320406/1139112749471444
- https://app.asana.com/0/856498667320406/1138820034182681

CC:

**Description**:
Removed notifications experiment.
Tweaks to copy of the Home Row CTA

**Steps to test this PR**:
1. Check if copy is changes for different variants.
2. Check Pixels for:
 - CTA show/no thanks actions
 - CTA reminder tap/no tap actions
 - Entering Instructions screen from settings.
 - Replaying video.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
